.github/workflows/builder-unit-test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Install dependencies
         shell: bash -l {0}
         run: |
-          conda install -y pylint=2.6.0 \
+          conda install -y pylint=2.7 \
```

Comment:
We'll need to keep an eye on issues that come up with specifying `2.7` instead of a specific `2.7.X`.